test/Confluent.Kafka.UnitTests/Headers.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+
+namespace Confluent.Kafka.UnitTests
+{
+    public class HeadersTests
+    {
+        [Fact]
+        public void GetLast()
+        {
+            var hdrs = new Headers();
+            hdrs.Add(new KeyValuePair<string, byte[]>("my-header", new byte[] { 42 }));
+            hdrs.Add(new KeyValuePair<string, byte[]>("my-header", new byte[] { 44 }));
+            hdrs.Add(new KeyValuePair<string, byte[]>("my-header-2", new byte[] { 45 }));
```

Comment:
Also test:
 * null value
 * empty vlaue
 * longer value than a single byte
 * utf8 string as value, encode + decode, since that is how most people will use this